server/src/main/java/gotcha/server/Utils/Logger/ServerLogger.java

```diff
@@ -10,6 +10,7 @@ public class ServerLogger {
 
     /*------------------------------------------------------ FIELDS ------------------------------------------------------*/
 
+    private final String logger_address = "C:\\Users\\amitm\\Desktop\\SemH\\ScooterGotcha\\Server\\logFiles\\ServerLog.txt";
```

Comment:
Maybe change it to be a Relative Path since this will only work on 1 computer each time

Reply:
yes i should do it. 
it is not working with relative path yet